src/Dropdown/Dropdown.js

```diff
@@ -70,7 +70,7 @@ type State = {
 };
 
 /**
- * Dropdown provides a transitory list of actions that appears upon user interaction with a trigger.
+ * Dropdown presents a list of options or actions after a user interacts with a trigger.
```

Comment:
Dropdown items are always “actions” (click to do something), never “options” (click to select that item).

Reply:
That was one of the minor details we were trying to resolve yesterday. Thanks for the clarification!

Reply:
updated.